workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -199,13 +209,28 @@ private static string GetClientLogFileFullPath()
             }
         }
 
-        public static void LaunchLocalDeployment()
+        public static void ConfigureLocalDeployment()
```

Comment:
I think this should still be called `Launch` as the function will launch a deployment, not just configure it.

Reply:
Since I've removed the local deployment with custom snapshot option the function returned to what it was